src/bootstrap/lib.rs

```diff
@@ -103,6 +103,8 @@
 //! More documentation can be found in each respective module below, and you can
 //! also check out the `src/bootstrap/README.md` file for more information.
 
+#![feature(once_cell)]
```

Comment:
Please switch to once_cell (the crate) for now -- we intentionally do not use unstable features in rustbuild, as it's a bootstrap tool.

Reply:
done